src/systems/ec/implementations/react/packages/radio/src/RadioGroup.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import RadioButton from './RadioButton';
+
+const RadioGroup = ({
+  labelId,
+  items,
+  name,
+  helperText,
+  label,
+  hideLabel,
+  invalid,
+  invalidText,
+  binary,
+  className,
+  ...props
+}) => {
+  const classNames = classnames(className, 'ecl-radio__group', {
+    'ecl-radio__group--invalid': invalid,
+    'ecl-radio__group--binary': binary,
+  });
+
+  return (
+    <div
+      {...props}
+      role="radiogroup"
+      aria-labelledby={labelId}
+      className={classNames}
+    >
+      {label && (
+        <div
+          className={classnames('ecl-form-label', {
+            'ecl-form-label--hidden': hideLabel,
+          })}
+          id={labelId}
+        >
+          {label}
+        </div>
+      )}
+
+      {helperText && (
+        <div className="ecl-radio__help ecl-help-block">{helperText}</div>
+      )}
+
+      {invalid && invalidText && (
+        <div className="ecl-radio__invalid ecl-feedback-message">
```

Comment:
What about adding aria-invalid attribute?
https://www.w3.org/TR/WCAG20-TECHS/ARIA21.html
What about aria-live to indicate screen reader that we changed the DOM (this is relevant in case of client side validation)
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions
https://hiddedevries.nl/en/blog/2017-04-04-how-to-make-inline-error-messages-accessible

Reply:
The aria-invalid is set on an input.
So on what input are we supposed to add it? every input radio in the group?

Reply:
aria-live should be handled in integration side, not on ECl, as we don't know how it willbe managed

Reply:
The "correct" way would be to always have a default value:
https://stackoverflow.com/questions/8509481/aria-required-in-a-radio-group
I'm getting back to ux/design on this